test/fork.js

```diff
@@ -72,6 +72,7 @@ test('fake timers do not break duration', function (t) {
 		});
 });
 
+/*
```

Comment:
With this change, destructuring would no longer work. This is not allowed anymore:

```js
test({is, same} => {
  is(1 + 1, 2);
  same([1,2].slice(1), [2]);
});
```

You have to preserve the this reference:

```js
test(t => {
  t.is(1 + 1, 2);
  t.same([1,2].slice(1), [2]);
});
```